backend/Perflow/Services/Implementations/SongsService.cs

```diff
@@ -36,10 +36,12 @@ public async Task<IEnumerable<SongReadDTO>> GetLikedSongsAsync(int userId)
                     .ThenInclude(song => song.Group)
                 .Include(songReaction => songReaction.Song)
                     .ThenInclude(song => song.Album)
-                .Select(songReaction => songReaction.Song)
+                .Select(songReaction =>
+                  mapper.Map<LikedSong, SongReadDTO>(new LikedSong(songReaction.Song, true))
```

Comment:
Add tab here, as we have here:
![image](https://user-images.githubusercontent.com/24195200/129344196-458feff8-c6a6-45c1-b89b-0f2ea410b2c1.png)